### DISCUSSION TITLE
q05_cell_abundance_w_sf represents the relative abundance of mRNA counts?

### DISCUSSION THREAD

**Discussion by Nasrine26** [BayraktarLab/cell2location#111] [Q&A] [answered]
Hello,
First of all, I really like your framework!

This may be a very stupid question but I wanted some clarifications on q05_cell_abundance_w_sf and q05_nUMI_factors. After cell2location model estimates cell abundances in each location, we get the absolute abundances of cell types (mean of the posterior distribution) and also the 5% quantile can retrieved (at least that amount is present). Do these abundances q05_cell_abundance_w_sf correspond to the relative amount of number of mRNA molecules confidently assigned to each cell type or the regression parameter w_sf (weight representing abundance of cell type in each spot)?

**Comment by Nasrine26** [comment, marked as answer]
Found the answer! 
q05_cell_abundance_w_sf  represents the regression parameter w_sf (5% quantile)

**Reply by vitkl** [reply]
Exactly, also see my response to your issue.

**Comment by epalo** [comment]
Hello!
Thanks for developing such a wonderful tool. 

Relating to this question, it is not clear to me what the meaning of this `q05_cell_abundance_w_sf`is. Is it relative or absolute abundance? What does it mean to have values ranging from 0 to 0,2 for a cell type? or 0 to 10? I don't understand as the maximum amount of cells by spot is 5. Could you explain a little more these amounts?

Thanks in advance!

**Reply by vitkl** [reply]
See https://discourse.scverse.org/t/further-explanation-of-q05-cell-abundance-w-sf-values/1231/2?u=vitkl
